tests/test_jax_utils.py

```diff
@@ -181,3 +182,23 @@ def test_pairwise_distances_periodic_lifting(coords_box_w):
     # guard assertion to prevent spurious failure due to roundoff error
     if np.std(w) / (1.0 + np.linalg.norm(x)) > 1e-6:
         assert dij.sum() > dij0.sum()
+
+
+def test_process_traj_in_chunks():
+    """process_traj_in_chunks should be a drop-in replacement for vmap"""
+
+    np.random.seed(2023)
+
+    T = 10000  # large num snapshots
+    N = 100
+
+    traj = np.random.randn(T, N, 3)
+    boxes = np.random.randn(T, 3, 3)
+
+    def f_snapshot(x, box):
+        return jnp.sum(x ** 2) + jnp.sum(box ** 3)  # arbitrary fxn
+
+    reference = vmap(f_snapshot)(traj, boxes)
+    actual = process_traj_in_chunks(f_snapshot, traj, boxes)
+
+    np.testing.assert_allclose(actual, reference)
```

Comment:
Should this test different `chunk_size`s? (including edge cases `chunk_size == 1`, `chunk_size == T`, and `T % chunk_size != 0`)

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/931/commits/bc584186a026e5859070a70c1d3291bff0f79baf

Reply:
Could this assert equality?

Reply:
Yep, strengthened in https://github.com/proteneer/timemachine/pull/931/commits/f06fee7ce1a1f4730c820032350275fb7ee54410